src/Solnet.Programs/Models/MultiSignatureAccount.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Solnet.Programs.Utilities;
+using Solnet.Wallet;
+using System;
+using System.Collections.Generic;
+
+namespace Solnet.Programs.Models
+{
+    /// <summary>
+    /// Represents a <see cref="TokenProgram"/> Multi Signature Account in Solana.
+    /// </summary>
+    public class MultiSignatureAccount
+    {
+        /// <summary>
+        /// The maximum number of signers.
+        /// </summary>
+        public const int MaxSigners = 11;
+
+        /// <summary>
+        /// The layout of the <see cref="MultiSignatureAccount"/> structure.
+        /// </summary>
+        public static class Layout
+        {
+            /// <summary>
+            /// The length of the structure.
+            /// </summary>
+            public const int Length = 355;
+
+            /// <summary>
+            /// The offset at which the number of signers required value begins.
+            /// </summary>
+            public const int MOffset = 0;
+
+            /// <summary>
+            /// The offset at which the number of valid signers value begins.
+            /// </summary>
+            public const int NOffset = 1;
+
+            /// <summary>
+            /// The offset at which the is initialized value begins.
+            /// </summary>
+            public const int IsInitializedOffset = 2;
+
+            /// <summary>
+            /// The offset at which the array with signers' public keys begins.
+            /// </summary>
+            public const int SignersOffset = 3;
+        }
+
+        /// <summary>
+        /// Number of signers required
+        /// </summary>
+        public byte M;
+
+        /// <summary>
+        /// Number of valid signers
+        /// </summary>
+        public byte N;
```

Comment:
Why not set M and N to something readable? 
e.g M -> MinimumSignatures, N -> NumberSignatures

Reply:
Done